src/colors.css

```
@@ -58,4 +58,69 @@ html[data-color-scheme="dark"] {
   color-scheme: dark;
 }
 
-/** TODO: Add colors for high contrast modes */
+html[data-color-scheme="light-high-contrast"], html:not([data-color-scheme]) {
```

```suggestion
html[data-color-scheme="light-high-contrast"] {
```

Copy & paste bug. This block shouldn't be active when there is no color scheme on the HTML element. That should only activate the "light" block above.
Same copy & paste bug for the block below.